### PR TITLE
Feature/yash member profile

### LINKED ISSUE #2564: [Community] Member Profile: Yash Kamboj
**Current Behavior**

@YashKamboj has been a consistent contributor and community member.

**Desired Situation**
Let's recognize @YashKamboj as a contributor and community member by creating a profile on https://layer5.io/community/members.

- GitHub: @YashKamboj 
- Twitter: @yashkamboj123
- LinkedIn: https://www.linkedin.com/in/yash-kamboj-9078011b5
- Link to profile picture:

A detailed explanation on how to set up a community member profile can be found in the [CONTRIBUTING.md](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)

---

**Contributor Resources**

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- See Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs). Join the [Layer5 Community](http://slack.layer5.io) for access.

### PR DESCRIPTION
**Pull request by KXLAA**
**Description**

This PR fixes #2564

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: src/collections/members/yash-kamboj/index.mdx [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+position: Contributor
+image_path: ./yash-kamboj.jpeg
+github: YashKamboj
+twitter: yashkamboj123
+linkedin: Yash Kamboj
+location: New Delhi, India
+bio: I am a undergrad student at IP university, Delhi. I am passionate about open-source and front-end development. Always up to learn new things. Love to help others and learning new thing from them.
+status: Active
+published: true
+---
```
Comment (verma-kunal) [inline src/collections/members/yash-kamboj/index.mdx:12]: @YashKamboj , could you pls confirm the details here!
Thanks

Comment (YashKamboj) [reply src/collections/members/yash-kamboj/index.mdx:12]: @KXLAA change the linkedin to yash-kamboj-9078011b5

Comment (YashKamboj) [reply src/collections/members/yash-kamboj/index.mdx:12]: Everything else, good, Great Work 👌

Comment (KXLAA) [reply src/collections/members/yash-kamboj/index.mdx:12]: alright

**Review by verma-kunal — APPROVED** [review]
Great work @KXLAA 🎉

**Comment by YashKamboj** [comment]
If you revert the package.json , everything is good @KXLAA

**Comment by KXLAA** [comment]
> 

revert? , that what state?

**Comment by YashKamboj** [comment]
@KXLAA  Just remove it from the commit , don't add package.json in the commits

**Comment by YashKamboj** [comment]
@KXLAA really sorry meant package-lock.json

**Comment by KXLAA** [comment]
> @KXLAA really sorry meant package-lock.json,

okay, i have deleted the package-lock.json

**Comment by YashKamboj** [comment]
@KXLAA remove from the commit! 😅, we don't want package-lock.json to merge

**Comment by KXLAA** [comment]
> 

hey sorry I'm confused, do I put the file in the gitignore or remove it entirely from my previous commits?

**Comment by YashKamboj** [comment]
@KXLAA remove it from your previous commits, if you face further problem open another pr , will be easier 😅

**Comment by warunicorn19** [comment]
Okay, I tried pretty hard to revert the package-lock but it didn't work. let's have a new PR @KXLAA

**Comment by l5io** [comment]
🚀 Preview for commit a7dbdb771134d720e9d164e356fc621e7deb653b at: https://61f9af553d09532611c6bfbb--layer5ng.netlify.app

**Comment by iambami** [comment]
@KXLAA can you give us a new PR?

**Comment by KXLAA** [comment]
> 

yes i would od that today, apologies

**Comment by leecalcote** [comment]
Merge conflict....

**Comment by warunicorn19** [comment]
This can be closed, another PR has already been merged.

**Pull request closed without merging**